app/views/figures/index.erb

```diff
@@ -0,0 +1,10 @@
+<table>
+  <th>Person Name</th>
+  <tr>
+    <% @figure.each do |figures| %>
```

Comment:
@figure is your variable name and `figures` is the block variable? I you sure it shouldn't be `@figures` is the instance variable and `figure` is the block variable?